pages/common/pr.md

```diff
@@ -0,0 +1,24 @@
+# pr
+
+> Paginate or columnate files for printing.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/pr-invocation.html>.
+
+- Print multiple files with a default header and footer:
+
+`pr {{file1 file2 file3}}`
+
+- Print all files in parallel, one in each column without a header or footer:
+
+`pr -m -T {{file1 file2 file3}}`
+
+- Print all files beginning at page 2 up to page 5 with a given page length (including header and footer):
+
+`pr +{{2}}:{{5}} -l {{page_length}} {{file1 file2 file3}}`
+
+- Print all files with an offset each line and a truncating custom page width:
```

Comment:
with an offset _for_ each line ..

And we can get rid of "all files". It is implicit after being mentioned 2 times above.